docs/stylesheets/style.css

```diff
@@ -632,11 +632,13 @@ p {
 @media only screen and (max-width: 768px) {
   .nav__items {
     flex-direction: column;
+    display: flex;
     position: fixed;
-    top: calc(calc(var(--gap-lg) * 2 + 32px) + 0.5rem);
+    top: 0;
```

Comment:
@joshreini1 this variable should be controlling where the first nav item starts to allow our logo and the close icon to be shown - I think we might want to keep it, but wanted to confirm with you

Reply:
Thanks! Tested and removed this change.